src/components/Layout/Header.js

```diff
@@ -14,6 +14,9 @@ const useStyles = makeStyles((theme) => ({
   logo: {
     flexGrow: 1,
   },
+  btn: {
+    marginRight: 10,
+  }
```

Comment:
Не соответсвует стилю prettier warning
```suggestion
  },
```

Reply:
исправлено